player/src/main.rs

```diff
@@ -336,8 +336,7 @@ impl GlobalExt for wgc::hub::Global<IdentityPassThroughFactory> {
                     .map(|vb| wgc::pipeline::VertexBufferLayoutDescriptor {
```

Comment:
is this the same type now as `wgt::VertexBufferLayoutDescriptor`? we should remove those - it's one of the benefits of doing the Rust-ification :)